content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md

```diff
@@ -0,0 +1,89 @@
+
+### Functions support for MySQL databases
+
+If you need to set up a MySQL schema that includes functions, procedures, and triggers, you can now create new databases that have that feature enabled. [To do this, see the “Options” documentation for RDS databases.](https://cloud.gov/docs/services/relational-database/#options) This allows you to build a wider range of applications on cloud.gov.
+
+### Improved instructions for running Drupal 8 for managing content
+
+A lot of government teams run Drupal for content management for their websites, including the [Drupal 8](https://www.drupal.org/8) version. With some modifications, Drupal runs well on cloud.gov. We’ve improved our [instructions and sample code](https://github.com/18F/cf-ex-drupal8) showing how to run Drupal in cloud.gov with best practices including configuration-as-code and continuous deployment principles. Contact our team if you need additional help!
+
+### Automatically label your applications with version control information
+
+Do you want to track which version of your code is running in each of your deployed applications? You can now use the cloud.gov API to automatically label your applications and other resources with your choice of metadata, such as the relevant git commit hash (SHA). For instructions, see [this guide to resource labeling](https://www.cloudfoundry.org/blog/labeling-cloud-foundry-api-resources-with-a-git-sha/).
+
+## Updated
+
+We encourage staying up to date with the latest Cloud Foundry Command-Line Interface (CLI) versions. The latest version is v6.46.0. [You can download the CF CLI update here.](https://github.com/cloudfoundry/cli/releases)
+
+We routinely update the operating system underneath your applications to ensure your applications are using the latest patched versions. For example, there was a high-severity Ubuntu Linux security vulnerability nicknamed [SACKPanic](https://wiki.ubuntu.com/SecurityTeam/KnowledgeBase/SACKPanic) that was announced on June 18, and we mitigated it by the evening of June 19.
+
+## Removed
+
+### Removed: CNAME ALIAS method for verifying DNS ownership
+
+When using the [CDN service](https://cloud.gov/docs/services/cdn-route/) to set up a custom domain name for an application, cloud.gov now only offers certificate provisioning via DNS challenges. We stopped supporting the CNAME ALIAS method. See the [updated steps for setting up DNS](https://cloud.gov/docs/services/cdn-route/#how-to-set-up-dns).
+
+### Removed: Experimental NFS (Network File System) volume service
+
+We are no longer offering our experimental Network File System (NFS) volume service. We are exploring better ways to offer persistent filesystem volumes, which would enable applications to store their state in a durable local filesystem.
```

Comment:
> We are exploring better ways to offer persistent filesystem volumes, which would enable applications to store their state in a durable local filesystem.

We aren't so this section should be removed.

Reply:
I clarified that we are looking into better persistent storage support, not necessarily filesystem.